pkg/configuration/configuration.go

```diff
@@ -91,6 +92,21 @@ func IsAuthCheckEnabled(cmd *cobra.Command) bool {
 	return true
 }
 
+func CheckMinAPIVersionRestriction(cmd *cobra.Command, currentVersion int) error {
+	c := cmd
+	for c != nil {
+		if c.Annotations != nil {
+			if s, ok := c.Annotations["MinAPIVersion"]; ok {
+				if minVersion, err := strconv.ParseInt(s, 10, 64); err == nil && int64(currentVersion) < minVersion {
+					return fmt.Errorf("Minimum API version %d is required to use the '%s' command. Current API version is %d", minVersion, c.Name(), currentVersion)
+				}
+			}
+		}
+		c = c.Parent()
```

Comment:
is this some debug leftover?

Reply:
No, this is what makes it recurse back up the command tree, so that we can set the annotation on the `service-users` command and it will apply to all sub commands as well.